### PR TITLE
fix: 10227 use dev config with modrun

### PR DESCRIPTION
**Pull request by jamesnguyentech**
Description:

This PR update modrun for running with the dev config.

Related issue(s):

Fixes https://github.com/hashgraph/hedera-services/issues/10227

### CODE REVIEW

**Review by nathanklick — CHANGES_REQUESTED** [review]

File: hedera-node/hedera-app/build.gradle.kts
Comment (jamesnguyentech) [reply hedera-node/hedera-app/build.gradle.kts]: I reverted it. Copy it back into the data folder as before, then copy the entire folder into build/node.

File: hedera-node/hedera-app/build.gradle.kts
Comment (nathanklick) [inline hedera-node/hedera-app/build.gradle.kts:158]:
```suggestion
        shouldRunAfter(tasks.named("copyApp"))
        shouldRunAfter(tasks.named("copyLib"))
```

**Review by nathanklick — DISMISSED** [review]

**Review by nathanklick — APPROVED** [review]

**Comment by nathanklick** [comment]
@jamesnguyentech Would you please run `./gradlew spotlessApply` on your PR branch to resolve the spotless check failure?

**Comment by jamesnguyentech** [comment]
@nathanklick I ran it. Could you check it again?

**Comment by nathanklick** [comment]
@jamesnguyentech Looks great, as soon as the remaining test run finishes I will merge unless there is an error. The unit test failure is a known issue which has already been fixed in the latest `develop` branch so I will bypass the failure.

**Pull request merged**
